Update Contact

sanjana-creoit opened the pull request:
- fix delete contact not updating contact list
- fix edit contact not updating in profile page

https://github.com/sanjana-creoit/figma_design/assets/142411868/a6d8530a-acec-4203-ba92-8e6e784e92e3

https://github.com/sanjana-creoit/figma_design/assets/142411868/ea6f1665-41f4-47c1-a521-54ccc8294f1a

sanjana-creoit committed:
Update Contact
- fix delete contact not updating contact list
- fix edit contact not updating in profile page

AnaghaVB commented:
The keyboard in Contact Page should not be shown unless you click on the Search, please fix this.

AnaghaVB commented:
When you click on a contact and navigate to Profile page, the loader for the name does not stop. When ever you open the Profile page, the name and number should be shown, the loader will appear only after you update the contact information

sanjana-creoit committed:
Update Profile Page
- fix keyboard in contactPage displaying unnecessarily
- fix loader in profilePage displaying before loading the value

AnaghaVB commented on lib/profilePage.dart:
```diff
@@ -81,16 +81,17 @@ class ProfilePage extends GetView<ProfilePageController> {
-                    ? CircularProgressIndicator()
-                    : Text(
+                child:
+                controller.isValueUpdate.value == null
```
Why are you checking for null?

sanjana-creoit committed:
Update Contact
- fix the issue with loader in profilePage
